### ISSUE TITLE
Editor rules on twitter tagging author

### ISSUE THREAD

**Issue by stefaniebutland** [ropensci-org/blog-guidance#44]
tl;dr Are you ok with author tagging rules being same as reviewers? 

In [5.3.4. Tagging Accounts](https://ropensci-blog-guidance.netlify.com/editor.html#tweetpromo)

> Only tag the account of an author if they included their username in their author file.

I understand the idea of not tagging someone on twitter if they don't want that, however, my experience is that authors don't pay much attention to their author file content, especially when we had created one automatically for them en masse.

I would prefer the author tagging rules be same as reviewers. 

> For reviewers, you might search for their account but only tag an account
it is active,
it is not anonymous,
it is at least partly used professionally by the owner. In case of doubt contact the person whose account you’d like to tag.

When an author's twitter meets the above criteria, I usually add their twitter to their author file

**Comment by maelle** [comment]
It sounds good!

**Comment by maelle** [comment]
But why not adding that as part of the review so the author sees its getting added?

**Comment by stefaniebutland** [comment]
added in Chapter 5 in https://github.com/ropensci-org/blog-guidance/pull/46
